app/src/main/java/org/kiwix/kiwixmobile/main/CompatFindActionModeCallback.java

```diff
@@ -43,13 +43,13 @@
 
   private View mCustomView;
 
-  private EditText mEditText;
```

Comment:
Why is this no longer private? If it is the synthetic accessor warning that should be deactivated in the project.
Also it would be nice if you would rename this to not be in Hungarian notation eg "m[VariableName]"

Reply:
Hi @macgills, I made the changes and committed the result.

Reply:
did you use `shift+f6` to rename?
MainActivity:600 ` else if (compatCallback.mIsActive)` 
It has not been renamed in other files causing a compilation error

Reply:
This also means you have been bypassing the git hooks? Please ensure lint is run on every commit

Reply:
No, I don't think I've been bypassing git hooks.  I haven't actively changed anything at least.  I believe it still runs lint.

Reply:
Then the commit would have failed because if it ran lint it would have built the app and it would have failed because that commit was not compilable. in the project folder you can look in `.git/hooks` and if `precommit` is there then the install was successful. If that file is there then you are bypassing commit hooks.

Reply:
Ok, the pre-commit file is indeed there.  Could you please let me know what to do to enable git hooks and/or lint?

Reply:
are you committing via Android Studio? there is a "bypass hooks" checkbox that I imagine is checked

Reply:
I'm using IntelliJ and the box is checked to run hooks.  I hadn't changed it when I pulled up the relevant screen, but I can't make a commit right now because there are no changes.  I will keep looking and try to find out what's going on.  Thank you for pointing out this problem.

Reply:
it could also be the commit hook is not installed? there should be `precommit` and `precommit.sample` in the hooks directory